application/models/file.go

```diff
@@ -92,6 +99,17 @@ func (f *File) Store(name string, content []byte) *FileUploadError {
 		return &e
 	}
 
+	// if possible, re-encode the image to strip EXIF metadata
+	img, _, err := image.Decode(bytes.NewReader(content))
+	if err == nil {
+		buf := new(bytes.Buffer)
+		if err := jpeg.Encode(buf, img, nil); err == nil {
+			content = buf.Bytes()
+			contentType = "image/jpg"
+			name = name + ".jpeg"
```

Comment:
Does the `name` being sent into this function already include the file extension? Will this result in `file.jpg.jpeg` and `file.png.jpeg`?

Reply:
Yes, also a lazy approach. It has the "benefit" of seeing the original format. Really won't be visible anywhere except S3 and possibly if a user asked the browser to "save image". Maybe better to just rename the file to `id + ".jpeg"`?